templates/login.html

```diff
@@ -1,2 +1,27 @@
-<!DOCTYPE html>
-<p><a href="{{ auth_url }}">Login</a></p>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+   <title>Spotipyn - Home Page</title>
+   <link rel="stylesheet" href="style.css">
+   <link rel="icon" type="image/png" href="favicon.png" sizes="32x32" />
+</head>
+<body>
+<table height="75" border="0" cellspacing="0" cellpadding="0">
+<div id="left">
+    <img src="logo.png" class="logo"/>
+</div>
+<div id="right">
+    <a href="<a href="{{ auth_url }}">Login</a>"><img src="button.png" class="button"/></a>

```

Comment:
I think this probably should be `<a href="{{ auth_url }}"><img src="button.png" class="button"/></a>`?